packages/fether-react/src/assets/sass/shared/_form.scss

```diff
@@ -110,6 +110,8 @@ input.form_field_amount[type='number'] {
   font-size: ms(6);
   line-height: ms(6) * 1.1;
   font-weight: 200;
+  padding: 3rem 0;
```

Comment:
Any reason for making the amount field so big?

Reply:
I was playing around with it while trying to address the tooltip blocking issue, and adding some padding made the screen felt a bit less suffocating. So was not much reasoning but more of a feeling :)

Reply:
From the few user interviews I've had so far, they really liked our design. I'd keep it this way (without the margin and padding) for now. We might need to add some space if we add some more info (like the amount in $), but that's the future :)